cost-analyzer/templates/cost-analyzer-frontend-config-map-template.yaml

```diff
@@ -54,7 +54,7 @@ data:
     upstream api {
 {{- if .Values.kubecostFrontend.useDefaultFqdn }}
         server {{ $serviceName }}.{{ .Release.Namespace }}.svc.cluster.local:9001;
-{{- else if and .Values.kubecostFrontend.api .Values.kubecostFrontend.api.fqdn }}
+{{- else if (.Values.kubecostFrontend.api).fqdn }}
```

Comment:
Thanks for this fix! I'm surprised the old logic doesn't work here?

I would've thought that Helm would try to evaluate whether `.Values.kubecostFrontend.api` exists (which it doesn't by default), then stop there before attempting to evaluate `.Values.kubecostFrontend.api.fqdn`. Regardless, thanks!

Reply:
I'm pretty sure that older versions of Helm did not have short-circuiting on the `and` operator